build/package/Installer.RPM.targets

```diff
@@ -0,0 +1,213 @@
+          SkipUnchangedFiles="False"
+          UseHardlinksIfPossible="False"/>
+
+    <Copy SourceFiles="@(AspNetRuntimeFilesInput)"
+          DestinationFiles="@(AspNetRuntimeFilesInput ->'$(rpmLayoutPackageRoot)/%(RecursiveDir)%(Filename)%(Extension)')"
+          OverwriteReadOnlyFiles="True"
+          SkipUnchangedFiles="False"
+          UseHardlinksIfPossible="False"/>
+
+    <!-- Replace config json variables -->
+    <ItemGroup>
+      <SDKTokenValue Include="%SHARED_HOST_RPM_VERSION%">
+        <ReplacementString>$(SharedFrameworkVersion)</ReplacementString>
+      </SDKTokenValue>
+      <SDKTokenValue Include="%SHARED_HOST_RPM_NAME%">
+        <ReplacementString>$(SharedFxRpmPackageName)</ReplacementString>
+      </SDKTokenValue>
+      <SDKTokenValue Include="%SDK_NUGET_VERSION%">
+        <ReplacementString>$(SdkVersion)</ReplacementString>
+      </SDKTokenValue>
+      <SDKTokenValue Include="%CLI_SDK_BRAND_NAME%">
+        <ReplacementString>$(SdkBrandName)</ReplacementString>
+      </SDKTokenValue>
+      <SDKTokenValue Include="%SDK_RPM_PACKAGE_NAME%">
+        <ReplacementString>$(SdkRpmPackageName)</ReplacementString>
+      </SDKTokenValue>
+    </ItemGroup>
+
+    <ItemGroup>
+      <TestSdkRpmTaskEnvironmentVariables Include="PATH=$(RpmInstalledDirectory)$(PathListSeparator)$(PATH)" />
+      <GeneratedInstallers Include="$(SdkInstallerFile)" />
+    </ItemGroup>
+
+    <ReplaceFileContents InputFile="$(ConfigJsonFile)"
+                         DestinationFile="$(rpmLayoutDirectory)$(rpmConfigJsonName)"
+                         ReplacementItems="@(SDKTokenValue)"/>
+
+    <!-- Call the task to build the pre-reqs (parameters, copyright, changelog) for calling the FPM tool -->
+    <BuildFPMToolPreReqs InputDir="$(rpmLayoutDirectory)"
+                         OutputDir="$(RpmIntermediatesDir)"
+                         PackageVersion="$(RpmPackageVersion)"
+                         ConfigJsonFile="$(rpmLayoutDirectory)$(rpmConfigJsonName)">
+      <Output TaskParameter="FPMParameters" PropertyName="FPMCmdParameters"/>
+    </BuildFPMToolPreReqs>
+
+    <!-- Build the RPM package by calling the FPM tool and passing the parameter list -->
+    <Exec Command="fpm $(FPMCmdParameters)" WorkingDirectory="$(RpmIntermediatesDir)"/>
+
+    <!-- Copy package to output -->
+    <ItemGroup>
+      <GeneratedRpmFiles Remove="@(GeneratedRpmFiles)"/>
+      <GeneratedRpmFiles Include="$(RpmIntermediatesDir)/*.rpm"/>
+    </ItemGroup>
+
+    <Error Text="@(GeneratedRpmFiles->Count()) .rpm files generated." Condition="'@(GeneratedRpmFiles->Count())' != 1"/>
+
+    <Copy SourceFiles="@(GeneratedRpmFiles)"
+          DestinationFiles="$(RpmFile)"
+          OverwriteReadOnlyFiles="True"
+          SkipUnchangedFiles="False"
+          UseHardlinksIfPossible="False"/>
+
+  </Target>
+
+  <Target Name="TestFPMTool">
+
+    <!-- run FPM  -->
+    <Exec Command="/usr/bin/env fpm --help > /dev/null" ContinueOnError="True">
```

Comment:
Sorry if this is a stupid question, but isnt `/usr/bin/env fpm` same as just calling `fpm`?